src/electron/views/automated-checks/automated-checks-view.tsx

```diff
@@ -70,17 +78,26 @@ export class AutomatedChecksView extends React.Component<AutomatedChecksViewProp
                 unifiedScanResultStoreData,
                 cardSelectionViewData.highlightedResultUids,
             );
+            reportHTML = deps.reportGenerator.generateFastPassAutomatedChecksReport(
+                deps.getCurrentDate(),
+                unifiedScanResultStoreData.targetAppInfo.name,
+                null,
```

Comment:
not for this PR (most likely): this `null` values here, does it make sense to rearrange the params and make this values optional or otherwise refactor how we pass data to this functions? I think the first null is related to the url, so I think you already have this in mind. Just leaving a comment so we keep this under our radar.

Reply:
No longer there 😄

Reply:
Actually still kind of there but yes, we need to refactor this in this/next week or so.